src/group/organizationGroup/organizationGroup.controller.ts

```diff
@@ -18,28 +20,101 @@ export class OrganizationGroup {
 
   static async getOrganizationGroups(cond?: Object): Promise<IOrganizationGroup[]> {
     const organizationGroups = await OrganizationGroup._organizationGroupRepository.find(cond);
-    const fieldsToSend = <(keyof IOrganizationGroup)[]> _.difference(ORGANIZATION_GROUP_KEYS, ORGANIZATION_GROUP_OBJECT_FIELDS);
+    const fieldsToSend = <(keyof IOrganizationGroup)[]>_.difference(ORGANIZATION_GROUP_KEYS, ORGANIZATION_GROUP_OBJECT_FIELDS);
     return _.flatMap(<IOrganizationGroup[]>organizationGroups, k => pick(k, ...fieldsToSend));
   }
 
+  /**
+   * Checks if there is a group with this hierarchy
+   * @param name Name of OrganizationGgroup
+   * @param hierarchy Hierarchy of OrganizationGgroup
+   */
   static async getOrganizationGroupByHierarchy(name: string, hierarchy: string[]): Promise<IOrganizationGroup> {
     const cond = {
       name,
       hierarchy,
     };
     const organizationGroup = await OrganizationGroup._organizationGroupRepository.findOne(cond);
-    if (!organizationGroup) return Promise.reject(new Error(`Cannot find group with name: ${name} and hierarchy: ${hierarchy}`));
+    if (!organizationGroup) return Promise.reject(new Error(`Cannot find group with name: ${name} and hierarchy: ${hierarchy.join('/')}`));
```

Comment:
It's totally fine to reject, I did it with "throw new error(...)" 
Maybe we should be consistent with style

Reply:
I don't change it now, we have to decide what we choose